spec/models/authority_spec.rb

```diff
@@ -1,6 +1,16 @@
 require 'spec_helper'
 
 describe Authority do
+  describe "validations" do
+    it 'should ensure a unique short_name' do
+      @a1 = create(:authority, short_name: "Existing")
+      @a2 = build(:authority, short_name: "Existing")
```

Comment:
If it starts getting re-used in more than one spot; I'll shift it to a context/before :each block... but it's only in one place right now.